components/homme/src/share/semoab_mod.F90

```diff
@@ -530,6 +532,7 @@ subroutine create_moab_meshes(par, elem, fv_nphys)
        ! first count the number of edges in the coarse mesh;
        ! use euler: v-m+f = 2 => m = v + f - 2
        nedges_c = nverts_c + nelemd - 1 ! could be more, if unconnected regions ?
+       nedges_c = nedges_c + 3 ! assume more than one connected region
```

Comment:
@rljacob @iulian787  did you add this? Is this for RRM? Euler formula should still hold as long as the meshes are conforming.